management-api-quick-start.md

```diff
@@ -114,7 +114,7 @@ The property `access_token` should be used for all other API requests in the `Au
 ### Step 3 - Get merchant sales units by organization number
 
 Send request
-[`GET:v1/merchants/{orgno}/sales-units`](https://developer.vippsmobilepay.com/api/management/#tag/Merchants/operation/getMerchantSalesUnits),
+[`GET:management/v1/merchants/{orgno}/sales-units`](https://developer.vippsmobilepay.com/api/management/#tag/Merchants/operation/getMerchantSalesUnits),
```

Comment:
Should be [[GET:/management/v1/merchants/{scheme}/{id}](https://developer.vippsmobilepay.com/api/management/#tag/Merchants/operation/getMerchant)](https://developer.vippsmobilepay.com/docs/APIs/management-api/management-api-guide/#get-the-sales-units-for-a-merchant-by-business-identifier)